### ISSUE TITLE
AttributeError: module 'eli5' has no attribute 'show_weights in Python 3.6

### ISSUE THREAD

**Issue by sean-huni** [TeamHG-Memex/eli5#282]
i'm getting this error in **Python 3.6. pip 18.1**

`AttributeError: module 'eli5' has no attribute 'show_weights'`

**Comment by lopuhin** [comment]
@sean-huni thanks for the report, this is quite weird I must say. Could you please help debugging this further? Can you paste the output of ``dir(eli5)`` and ``print(eli5)``?

**Comment by sean-huni** [comment]
@lopuhin You're welcome. 
```
print(dir(eli5))

['__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__path__', '__spec__', '__version__', '_decision_path', '_feature_importances', '_feature_names', '_feature_weights', '_graphviz', 'absolute_import', 'base', 'base_utils', 'explain', 'explain_prediction', 'explain_prediction_df', 'explain_prediction_dfs', 'explain_prediction_sklearn', 'explain_weights', 'explain_weights_df', 'explain_weights_dfs', 'explain_weights_sklearn', 'format_as_dataframe', 'format_as_dataframes', 'format_as_dict', 'format_as_html', 'format_as_text', 'format_html_styles', 'formatters', 'permutation_importance', 'sklearn', 'transform', 'transform_feature_names', 'utils']
```

And 
```
print(eli5)
<module 'eli5' from '/dev_0/proj/deepl/tensorflow/tens-env/lib/python3.6/site-packages/eli5/__init__.py'>
```

Does that help?

**Comment by lopuhin** [comment]
@sean-huni ah I see now, thanks for you help!

I think that the reason ``show_weights`` is not available, is that it is intended to be used in the notebook environment, so won't be available without ``IPython`` module installed.

**Comment by sean-huni** [comment]
Thank you @lopuhin. I was working directly from the Linux terminal. Sweet, I will now switch to the IPython Notebook. I'm sure that should solve the problem as you've mentioned.
